src/ceph-create-keys

```diff
@@ -237,7 +235,7 @@ def bootstrap_key(cluster, type_):
                 else:
                     raise
     if wait_count == 0:
-        raise SystemExit("Could not get or create %s bootstrap key after 10 minutes" % type_)
+        raise SystemExit("Could not get or create %s bootstrap key after %d seconds" % type_, wait_count)
```

Comment:
I don't think this will work? 

    Traceback (most recent call last):
      File "error.py", line 4, in <module>
        raise SystemExit("Could not get or create %s bootstrap key after %d seconds" % type_, wait_count)
    TypeError: not enough arguments for format string

I think this would need to be:

    raise SystemExit("Could not get or create %s bootstrap key after %d seconds" % (type_, wait_count))